docker/docker-compose.preview.yml

```diff
@@ -0,0 +1,116 @@
+version: '3.8'
+volumes:
+    node_modules:
+
+# Okteto specific
+endpoints:
+    - path: /
+      service: lightdash-preview
+      port: 8080
+
+services:
+    minio:
+        image: bitnami/minio:latest
+        ports:
+            - '9000'
+            - '9001'
+        environment:
+            - MINIO_ROOT_USER=minioadmin
+            - MINIO_ROOT_PASSWORD=${S3_SECRET_KEY}
+            - MINIO_DEFAULT_BUCKETS=results
+
+    db-preview:
+        image: pgvector/pgvector:pg16
+        restart: always
+        environment:
+            - POSTGRES_PASSWORD=${PGPASSWORD}
+        ports:
+            - '5432'
+
+    headless-browser:
+        image: ghcr.io/browserless/chromium:v2.24.3
+        restart: always
+        ports:
+            - '3000'
+
+    lightdash-preview:
+        build:
+            target: pr-runner
+            context: ..
+            dockerfile: dockerfile-prs
+        depends_on:
+            - db-preview
+            - headless-browser
+        healthcheck:
+            interval: 10s
+            timeout: 5s
+            retries: 10
+            start_period: 30s
+            http:
+                path: /api/v1/health
+                port: 8080
+        environment:
+            # these were set in the dockerfile-prs container
+            NODE_ENV: production
+            LIGHTDASH_MODE: pr
+            DBT_DEMO_DIR: /usr/app
+            INTERNAL_LIGHTDASH_HOST: http://lightdash-preview:8080
```

Comment:
this :8080 might be wrong here, since this will be pointing to the backend, not the frontend, right ?